src/ContributionsFromRenee/SokoGoal.js

```diff
@@ -0,0 +1,12 @@
+//copied from ./entity/SokoGoal.js
+import 'phaser';
```

Comment:
Do you need to import phaser in every file? Couldn't you just import it once at the root?